src/xcode/ENA/ENA/Source/Views/EmptyStateView/EmptyStateView.swift

```diff
@@ -83,7 +83,7 @@ class EmptyStateView: UIView {
 		stackView.addArrangedSubview(descriptionLabel)
 
 		topConstraint = containerView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor)
-
+		let percentageWidth = UIScreen.main.bounds.width / 3
```

Comment:
where does this magic number 3 come from? Maybe add a comment.

Reply:
I searched for a divider that fits for all four occurrences for the EmptyStateView (Certificates, Checkins, Contact Diary, Trace Locations).
But yes, good point, i will leave a comment.